lib/ash/error/invalid/page_requires_pagination.ex

```diff
@@ -0,0 +1,16 @@
+defmodule Ash.Error.Invalid.PageRequiresPagination do
+  @moduledoc "Used when page option is passed but pagination is not enabled."
+  use Ash.Error.Exception
+
+  def_ash_error([:resource, :action], class: :invalid)
+
+  defimpl Ash.ErrorKind do
+    def id(_), do: Ash.UUID.generate()
+
+    def code(_), do: "page_requires_pagination"
+
+    def message(%{resource: resource, action: action}) do
+      "Pagination is not enabled on resource #{inspect(resource)} for the action #{inspect(action)}. Check that you've set pagination to `true` in your action and also that you have set a page in your opts like this MyApp.MyAPI.read!(MyResource, page: [limit: 1])"
```

Comment:
```suggestion
      """
      Pagination is not enabled on resource #{inspect(resource)} for the action #{inspect(action)}. Check that you've
      enabled pagination in your action. For example:
      
      
      read #{action.name} do
        pagination offset?: true, keyset?: true, required?: false
      end
      """
```

Reply:
Making the error multiline will make it read a bit better and adding an example of how to fix it will be useful :D

Reply:
Pushed this. Thank you for the suggestion!

Reply:
> Making the error multiline will make it read a bit better and adding an example of how to fix it will be useful :D

Agreed!